tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/Test_SphericalTorus.cpp

```diff
@@ -157,6 +187,7 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.AnalyticData.GrMhd.SphericalTorus",
           const auto numerical_hessian = numerical_derivative(
               [&i, &j, &partial_torus](const std::array<double, 3>& y) {
                 const auto y_tnsr = a2t(y);
+                CAPTURE(y);
```

Comment:
This doesn't do anything.  There are no checks before it goes out of scope.

Reply:
done.